spec/ParseObject.spec.js

```diff
@@ -2055,4 +2055,33 @@ describe('Parse.Object testing', () => {
     const object = new Parse.Object('CloudCodeIsNew');
     await object.save();
   });
+
+  fit('should can custom objectId', async () => {
+    await reconfigureServer({
+      allowCustomObjectId: true,
+    });
+
+    Parse.allowCustomObjectId = true;
+
+    const body = {
+      name: 'SaveAudio',
+      data: '{"objectId":"66541dbdffead844f1339e3c","createdAt":"2024-05-27T05:44:29.504Z","updatedAt":"2024-05-27T05:44:29.504Z"}',
```

Comment:
Why are you setting the createdAt date here? Could you just create a new object, set a custom object Id and save it, without parsing a JSON. Remember, a test should be as simple as possible to demonstrate the issue.